WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -80,14 +80,16 @@ - (RemoteUser *)remoteUserFromDictionary:(NSDictionary *)dictionary
 
 - (NSArray *)remoteBlogsFromJSONArray:(NSArray *)jsonBlogs
 {
-    NSMutableArray *remoteBlogs = [NSMutableArray arrayWithCapacity:[jsonBlogs count]];
-    for (NSDictionary *jsonBlog in jsonBlogs) {
-        BOOL isJetpack = [jsonBlog[@"jetpack"] boolValue];
-        if (!isJetpack || JetpackREST.enabled) {
-            [remoteBlogs addObject:[self remoteBlogFromJSONDictionary:jsonBlog]];
-        }
+    NSArray *blogs = jsonBlogs;
+    if (!JetpackREST.enabled) {
```

Comment:
The conditions i had in mind are:

`JetpackREST.enabled = true` +  `jsonBlog[@"jetpack"] = false` (for a random blog) 

Old algorithm would have included that Blog into the `remoteBlogs` array, while the new one wouldn't even enter into the `if` in line 84.

(Sorry if this is incorrect, maybe i need more coffee?)